contrib/test-suite/test-suite.yml

```diff
@@ -1,12 +1,23 @@
-- name: Integration Tests
+# - name: Integration Tests
```

Comment:
Is this ("#") accidental commit of test stuff?

Reply:
I'm still testing it so I disabled integration test to make it run faster. I'll set waiting for review once it is ready.